HW02/Pavel_Karenda/pavel_karenda_hw_02_t_03.rb

```diff
@@ -0,0 +1,50 @@
+require 'time_difference'
+require 'time'
+
+CORE = /Calling core with action/.freeze
+DATE = /^([0-9]*-[0-9]*-[0-9]* [0-9]*:[0-9]*:[0-9]*.[0-9])/.freeze
+arr_lines = []
+
+if ARGV.length != 1
+  puts 'We need exactly one parameter. The name of a file.'
+  exit
+end
+filename = ARGV[0]
+puts "Going to open '#{filename}'"
+
+fh = File.open(filename)
+
+fh.each do |line|
+  arr_lines.push(line) if line.match(CORE)
+end
+fh.close
+
+def input_start_validity(arr)
+  return unless arr.length.zero?
+
+  p 'input has no paramethers'
+  exit
+end
```

Comment:
It is bad practice to combine logic and conclusion. Make a method to test and rewrite the method
```suggestion
def input_start_validity(arr)
  (arr.length.zero?) ? YOUR_VALIDATE_METHOD : return
end
```

Reply:
I hope I understood correct the idea.